portal/config/model_persistence.py

```diff
@@ -99,6 +100,7 @@ def import_(self, keep_unmentioned):
                 query.delete(synchronize_session=False)
 
         self.update_sequence()
+        trace("Import of {} complete".format(self.model.__name__))
```

Comment:
Why are you capturing these "log" messages in a trace instead of logging?

Reply:
that one is just noise when calling `flask seed` - it's extra that i wanted to view when calling live from the /settings page.  otherwise, you'll note the use of the common `_log()` method.